CMakeLists.txt

```diff
@@ -1373,23 +1373,19 @@ if(UNIX AND NOT APPLE)
   if (INSTALL_USER_UDEV_RULES)
     install(
       FILES
-        "${CMAKE_CURRENT_SOURCE_DIR}/res/linux/mixxx-usb-uaccess.rules"
+        "${CMAKE_CURRENT_SOURCE_DIR}/packaging/linux/69-mixxx-usb-uaccess.rules"
       DESTINATION
-        "${MIXXX_INSTALL_DATADIR}/udev/rules.d"
+        "${CMAKE_INSTALL_LIBDIR}/udev/rules.d"
     )
     install(CODE "
     message(STATUS \"Important Note: Installation of udev rules\n\"
         \"The udev rule file for USB HID and Bulk controller permissions will be\n\"
         \"installed to:\n\"
-        \"    ${CMAKE_INSTALL_PREFIX}/${MIXXX_INSTALL_DATADIR}/udev/rules.d.\n\"
+        \"    ${CMAKE_INSTALL_LIBDIR}/udev/rules.d.\n\"
         \"If you are installing Mixxx from source for your own use, copy\n\"
         \"mixxx-usb-uaccess.rules to /etc/udev/rules.d/ and run:\n\"
         \"    udevadm control --reload-rules && udevadm trigger\n\"
-        \"as root to load the rules.\n\"
-        \"If you are building a package for a distribution, the correct directory for\n\"
```

Comment:
Because the intent of this PR is to install it to the right place in the first place. At that point this comment is unnecessary, it'll already be in the right place.

Reply:
Once we install it at the right place, we need to rewrite this message. Maybe it can be omitted entirely and only be issues as a fatal message if the detection of the correct install directory fails. 
For that case we may take the directory including file name as a cmake option.